src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-multisite-zone-form/rgw-multisite-zone-form.component.html

```diff
@@ -119,15 +119,18 @@
         </label>
         <div class="cd-col-form-input">
           <input class="form-control"
-                 type="text"
+                 type="password"
```

Comment:
when you change the type to password, you'll need to provide the eye icon to show the password.

Reply:
You can check the references in the password field that we normally use across dashboard

Reply:
Updated the PR